README.md

```diff
@@ -4,6 +4,12 @@
   </a>
 </p>
 
+<p align="center">
+  <a href="https://cloud.digitalocean.com/apps/new?repo=https://github.com/cfu288/fasten-onprem/tree/add-deploy-to-digital-ocean-button">
```

Comment:
href needs to be updated to `https://github.com/fastenhealth/fasten-onprem/tree/main` before deploy